src/config-schema.ts

```diff
@@ -41,8 +41,13 @@ export const configSchema = {
     _default: '05a29f94-c0ed-11e2-94be-8c13b969e334',
     _description: 'Needed if the "id" column of "labTableColumns" is used. Is the OpenMRS ID by default.',
   },
+  enableReviewingLabResultsBeforeApproval: {
```

Comment:
I think this should be left as `true` by default as that basically replicates the behaviour today. Since it's a configuration option, it's configurable.

Left out of this is that the whole "do we use a result approval" workflow itself seems like it probably should be configurable.